### PR TITLE
docs: Add CHANGELOG.md

### PR DESCRIPTION
**Pull request by kyliau**
This will show up in vscode to inform users of changes and new features.

Screenshot from 2019-10-17 11-35-33

### CODE REVIEW

**Comment by kyliau** [comment]
Did I miss anything significant in the changelog?

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# v0.900.0
+This is a major overhaul that brings signficant performance improvements.
```
Comment (ayazhafiz) [inline CHANGELOG.md]:
```suggestion
This is a minor overhaul that brings signficant performance improvements.
```

Just to make sure that we stick with semantic versioning.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+New features:
+- Added "go to definition" for `templateUrl` and `styleUrls`.
+- Hover tooltip now shows the `NgModule` a directive is declared in.
+- Added `angular.ngdk` config for specifying location of `@angular/language-service`.
+- Added vscode command to restart the extension.
+- Display loading indicator while project is loading.
```
Comment (ayazhafiz) [inline CHANGELOG.md]: Maybe include that the language server is published as a separate package? Perhaps we should have a separate changelog for that?

Comment (ayazhafiz) [inline CHANGELOG.md]: Is the workspace/bundled/user-specified `@angular/language-service` a new change in this version?

File: CHANGELOG.md (hunk 1 above)
Comment (kyliau) [reply CHANGELOG.md]: I meant to say "major" as in "substantial". Reworded the sentence 😉

File: CHANGELOG.md (hunk 2 above)
Comment (kyliau) [reply CHANGELOG.md]: yeah, I'll create another changelog for the server package.
This one is meant for the vscode extension, so I don't think it's worthwhile mentioning the new server package here.

**Pull request merged**